css-layout-classes/styles.css

```diff
@@ -0,0 +1,51 @@
+* {
+  font-family: sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-third {
+  width: 100%;
+  flex-wrap: nowrap;
```

Comment:
`flex-wrap` is a parent property, which means it gets applied where `display: flex` gets applied, so this isn't doing anything and should be deleted